### PR TITLE
Logout button

### PR DESCRIPTION
**Pull request by lbkchen**
Pls merge #19 before this. Then I will merge the master in 

* Adds a logout button on the profile page that signs out and redirects user to login
logout

### CODE REVIEW

**Comment by lbkchen** [comment]
#25

**Comment by lbkchen** [comment]
@nikitr fixed merge conflicts

File: src/helpers/requesters/BaseRequester.js [hunk 1]
```diff
@@ -63,8 +63,12 @@ class BaseRequester {
 
     return fetch(endpoint, payload).then((response) => {
       if (!response.ok) { throw response; }
+      if (response.status === 204) { return {}; }
+      console.log(response);
       return response.json();
     }).then((json) => {
+      console.log("JSON");
```
Comment (nikitr) [inline src/helpers/requesters/BaseRequester.js]: remove console log

**Comment by lbkchen** [comment]
@nikitr ready for review

**Pull request merged**